application/common/mail/method/verify.php

```diff
@@ -0,0 +1,15 @@
+Hi there,
+
+Someone recently requested to add this email address, <?php echo \yii\helpers\Html::encode($toAddress); ?>,
+as a method for verifying themselves should they need to reset their <?php echo \Yii::$app->params['idpName']; ?>
+account. If this was you, you can use the reset code below to set a new password.
+
+Reset Code: <?php echo \yii\helpers\Html::encode($code); ?>
+
+If you did not request adding this email address to your account please delete this email.
+
+To keep your account secure, please don't forward this email to anyone.
+See our Help Center for <a href="<?php echo \Yii::$app->params['helpCenterUrl']; ?>">more security tips</a>.
```

Comment:
The helpCenterUrl value might also need to be escaped, so that any special characters (such as ampersands) are properly encoded for use in an HTML attribute.

Just from a separation of concerns standpoint, it would probably also be better if the helpCenterUrl was passed into this view rather than having this view reach out for it. We recently had to fix a bug in another project that was partially caused by a view file reaching out to somewhere else in the code rather than being given all the data that it needed.